standard/ERCs/erc-6900.md

```diff
@@ -44,9 +44,7 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
   - **Validation functions** validate authorization on behalf of the account.
   - **Execution functions** execute custom logic allowed by the account.
   - **Hooks** execute custom logic and checks before and/or after an execution function or validation function.
-There are two types of hooks:
-    - **Validation hook** functions run before a validation function. These can enforce permissions on actions authorized by a validation function.
-    - **Execution hook** functions run can run before and/or after an execution function. The pre execution hook may optionally return data to be consumed by a post execution hook functions.
+    There are two types of hooks: - **Validation hook** functions run before a validation function. These can enforce permissions on actions authorized by a validation function. - **Execution hook** functions run can run before and/or after an execution function. The pre execution hook may optionally return data to be consumed by a post execution hook functions.
```

Comment:
```suggestion
    There are two types of hooks: - **Validation hook** functions run before a validation function. These can enforce permissions on actions authorized by a validation function. - **Execution hook** functions run before and/or after an execution function. Pre execution hook functions may optionally return data to be consumed by corresponding post execution hook functions.
```

Reply:
Let me revert this. This is not supposed to be included and is not by me. SOOO weird. I am gonna have to dig into this cause it keeps auto changed...

Reply:
Fixing the typos and grammar tho.

Reply:
Good stuff!